packages/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -24,4 +122,61 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
       reportExpirySeconds: await this.reportExpirySeconds(),
     }
   }
+
+  /**
+   * Helper function to get the rates for StableToken, by passing the address
+   * of StableToken to `getRates`.
+   */
+  getStableTokenRates = async (): Promise<OracleRate[]> => this.getRates(CeloContract.StableToken)
+
+  /**
+   * Gets all elements from the doubly linked list.
+   * @param token The CeloToken representing the token for which the Celo
+   *   Gold exchange rate is being reported. Example: CeloContract.StableToken
+   * @return An unpacked list of elements from largest to smallest.
+   */
+  async getRates(token: CeloToken): Promise<OracleRate[]> {
```

Comment:
Here (and idk if applies to other methods) we should divide by DENOMINATOR. 
The idea is that from the perspective of contractkit user, there's no need to know that DENOMINATOR even exists.

So if i report `1/2` as a rate, i would expect to do `getRates()` and find `0.5`

Reply:
That makes sense. However, it seems like we go to great lengths to never have a decimal value and express everything as the relationship between BigNumbers to avoid the loss of precision that occurs when JS touches a decimal value.

Would a BigNumber decimal be a safe choice here?

Reply:
sorry, i wasn't clear :D yes. the idea is to return a BigNumber, always, but that bignumber should represent 0.5 in my expample. (Bignumber has decimal positions, contrary to solidity)